packages/docs-reanimated/docs/fundamentals/getting-started.mdx

```diff
@@ -32,7 +32,7 @@ If you don't have an existing project, you can create a new Expo app using a tem
   </TabItem>
 </Tabs>
 
-Alternatively, you can dive into [our examples](https://github.com/software-mansion/react-native-reanimated/tree/3.3.0/app/src/examples) on GitHub.
+Alternatively, you can dive into [our examples](https://github.com/software-mansion/react-native-reanimated/tree/3.14.0/apps/common-app/src/examples) on GitHub.
```

Comment:
```suggestion
Alternatively, you can dive into [our examples](https://github.com/software-mansion/react-native-reanimated/tree/main/apps/common-app/src/examples) on GitHub.
```
what do you think? 🤔

Reply:
im not a fan of linking to main
eg. when we changed to monorepo the link was still working correctly because it was anchored to a tag